api/v1alpha1/dspipeline_types.go

```diff
@@ -80,6 +80,13 @@ type APIServer struct {
 	// for the api server to use instead.
 	CustomServerConfig *ScriptConfigMap `json:"customServerConfigMap,omitempty"`
 
+	// Include custom kfp-launcher ConfigMap with the deployment of this DSP API Server. Default: false
+	// +kubebuilder:validation:Optional
+	EnableCustomKfpLauncher bool `json:"enableCustomKfpLauncher"`
```

Comment:
rather than introduce a switch AND custom config item, why don't we just assume based on if the Config exists or not

Reply:
Removed the boolean as mentioned and now I am just check for the existence of custom config map